apisix/plugins/limit-conn/init.lua

```diff
@@ -64,10 +64,9 @@ function _M.increase(conf, ctx)
     end
 
     if key == nil then
-        core.log.info("bypass the limit conn as the key is empty")
-        -- Bypass the limit conn when the key is empty.
-        -- This behavior is the same as Nginx
-        return
+        core.log.info("The key of limit conn is empty or invalid, set $remote_addr as the key")
+        -- When the key is empty or invalid, set $remote_addr as the default key
+        key = "remote_addr"
```

Comment:
The key should be the value of remote_addr not remote_addr itself

Reply:
Done. Thanks